pkgs/development/libraries/libvisual/default.nix

```
@@ -1,51 +1,35 @@
 { lib
 , stdenv
-, fetchurl
+, fetchFromGitHub
 , fetchpatch
 , SDL
 , autoreconfHook
+, autoconf-archive
 , glib
 , pkg-config
 }:
 
 stdenv.mkDerivation rec {
   pname = "libvisual";
-  version = "0.4.1";
+  version = "0.4.2";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/libvisual/${pname}-${version}.tar.gz";
-    hash = "sha256-qhKHdBf3bTZC2fTHIzAjgNgzF1Y51jpVZB0Bkopd230=";
+  src = fetchFromGitHub {
+    owner = "Libvisual";
+    repo = pname;
+    rev = "${pname}-${version}";
+    hash = "sha256-bDnpQODXB2Z6hezVoh7c6cklp6qpyDzVBAnwZD8Gros=";
   };
 
-  outputs = [ "out" "dev" ];
+  sourceRoot = "${src.name}/${pname}";
```

sourceRoot should not have a reference to pname
```suggestion
  sourceRoot = "${src.name}/libvisual";
```